_pytest/test_sanity.py

```diff
@@ -0,0 +1,66 @@
+import importlib
+import pkgutil
+
+import pytest
+from six import PY2
+
+
+def import_submodules(package):
+    """ Import all submodules of a module, recursively, including subpackages."""
+
+    if isinstance(package, str):
```

Comment:
This is problematic in python2 since `unicode` is not an instance of `str`. In python2 there's basestring, but it doesn't exist in python3, hehe. This is how we are supposed to do it:
```
from six import string_types
isinstance(s, string_types)
```